release/release_gold.sh

```diff
@@ -94,6 +94,8 @@ function main {
 
 	#lc_time_run prepare_next_release_branch
 
+	lc_time_run update_release_info_date
```

Comment:
@kiwm, shouldn't we also re-enable the function `prepare_next_release_branch`?

Better! Can you create a test to it in order to re-enable it?